paddle/fluid/operators/detail/grpc_server.h

```diff
@@ -129,14 +100,16 @@ class AsyncGRPCServer final {
 
   std::unique_ptr<std::thread> t_prefetch_;
 
-  std::unique_ptr<framework::ExecutorPrepareContext> prefetch_ctx_;
-  framework::ProgramDesc *program_;
-  framework::Executor *executor_;
   int selected_port_;
 
   std::mutex mutex_ready_;
   std::condition_variable condition_ready_;
   int ready_;
+
+  GRPCProcessorCtx* rpc_processor_;
+
+  // barrier
```

Comment:
Need more details about this comment, it's confusion with only one word `barrier`.

Reply:
Change a readable name.
Thanks! Done!